common/djangoapps/philu_commands/management/commands/send_referral_follow_up_email.py

```diff
@@ -5,35 +5,40 @@
 from django.utils import timezone
 
 from openedx.features.smart_referral.models import SmartReferral
-from openedx.features.smart_referral.tasks import task_send_referral_follow_up_email
+from openedx.features.smart_referral.tasks import task_send_referral_follow_up_emails
 
 CONTACT_EMAIL = 'contact_email'
```

Comment:
We don't need this constant anymore since 'contact_email' is now used only once in this file, and the string itself is self-explanatory.

Reply:
Still we are avoiding string, I guess there's no harm in doing this.

Reply:
What I remember in clean code sessions we read that use magic constants for things which are not clear .. like 3 we don't know what would be 3 ..so it's better to use constant for 3 .. 
but contact_email is self explanatory string.. so i think we can avoid this constant..

Reply:
Secondly constant `CONTACT_EMAIL` and string `'contact_email'` are almost same.. So does this really benefit in terms of reusability even if it would be used on multiple places?

Reply:
@zeshanarifios Please review, [https://docs.google.com/document/d/11jqz9wpX4twsrFy5k6jlAazO7oTeQPqr415ZojmBAFM/edit#heading=h.fj7jz7a37jgn](Code review checklist)